opensrp-chw-core/src/test/java/org/smartregister/chw/core/activity/CoreStockInventoryItemDetailsReportActivityTest.java

```diff
@@ -54,9 +54,18 @@ public void setUp() {
         //context.session().setPassword(password);
 
         String stockItemName = "Male Condoms";
+        String stockItemName2="Female Condoms";
+        String stockItemName3="Cycle beads (Standard day method)";
+        String stockItemName4="Paracetamol";
+        String stockItemName5="";
+
         String providerName = "All-CHWs";
         Intent intent = new Intent();
         intent.putExtra(CoreConstants.HfStockUsageUtil.STOCK_NAME, stockItemName);
+        intent.putExtra(CoreConstants.HfStockUsageUtil.STOCK_NAME, stockItemName2);
+        intent.putExtra(CoreConstants.HfStockUsageUtil.STOCK_NAME, stockItemName3);
+        intent.putExtra(CoreConstants.HfStockUsageUtil.STOCK_NAME, stockItemName4);
+        intent.putExtra(CoreConstants.HfStockUsageUtil.STOCK_NAME, stockItemName5);
```

Comment:
It looks like the idea is to test     [private String evaluateStockName(String stockName) {](https://github.com/opensrp/opensrp-client-chw-core/blob/6b91ea622f7126923ab873371250566cf0610a56/opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreStockInventoryItemDetailsReportActivity.java#L38)

I'd suggest writing a different test because while we use `putExtra()` to Add extended data to the intent when starting an Activity for example, we use unique name/key -> value pairs since in https://github.com/opensrp/opensrp-client-chw-core/blob/master/opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreStockInventoryItemDetailsReportActivity.java#L92 it reads only one value.

Reply:
Maybe if you have a way to be able to write the test, I would be happy to get some assistance